puppeteer/spec/request-blocking.spec.ts

```diff
@@ -0,0 +1,64 @@
+import puppeteer, { HTTPResponse } from 'puppeteer';
+import { USER_LIST_SELECTOR, OUTPUT_RESPONSE } from '../data/selectors.json';
+import { BASE_URL, USER_LIST_URL } from '../data/constants.json';
+import { expect } from 'chai';
+import { resolve } from 'path';
+
+describe('Request Blocking', () => {
+  describe('Blocking the request', () => {
+    let browser: puppeteer.Browser;
+    let page: puppeteer.Page;
+    let errorText: string;
+    let intervalId: NodeJS.Timer;
```

Comment:
This variable could be declared inside the `before` hook, as it is not used anywhere else.